tests/e2e/pageObjects/components/overview-panel.ts

```diff
@@ -35,4 +41,11 @@ export class OverviewPanel {
     async verifyDbIndexSelected(dbIndex: number): Promise<void> {
         await t.expect(this.changeIndexBtn.textContent).contains(dbIndex.toString());
     }
+
+    /**
+     * wait for cpu is displayed
+     */
+    async waitForCpuIsCalculated(): Promise<void> {
+        await t.expect(this.overviewSpinner.visible).notOk();
```

Comment:
can you add some error message here?
and it's better to use 'exists' because visible+notOk will wait for extra ~5sec

Reply:
will try, thanks